packages/vue-instantsearch/src/components/__tests__/__snapshots__/RefinementList.js.snap

```diff
@@ -46,6 +47,7 @@ exports[`allows search bar classes override when it's searchable 1`] = `
              role="img"
              viewbox="0 0 20 20"
              width="1em"
+             aria-hidden="true"
```

Comment:
these Vue snapshots need aria-hidden as the first property (it's alphabetically sorted)

Reply:
@Haroenv Thanks for pointing that out! I've made the necessary adjustments. However I can't seem to get past that flaky unit test on `packages/react-instantsearch-hooks-server/src/__tests__/modules-none.test.tsx`